.github/workflows/publish.yml

```diff
@@ -14,12 +14,10 @@ jobs:
       release_created:  ${{ steps.release.outputs.release_created }}
       release_tag:  ${{ steps.release.outputs.tag_name }}
     steps:
-      - uses: google-github-actions/release-please-action@v3
+      - uses: google-github-actions/release-please-action@v4
         id: release
         with:
           release-type: node
-          package-name: corepack
-          bump-minor-pre-major: true # TODO: remove this when releasing v1.0.0.
```

Comment:
Removing this will likely be annoying, unless we are to actually release v1.0.0. The alternative is to setup a manifest, but the docs imply it's not a very straight forward thing to do, so maybe it's best to hold on with this update for the time being.